src/SharedMath/Colors/RgbaColor.cs

```diff
@@ -0,0 +1,25 @@
+namespace WpfMath.Colors
+{
+    public struct RgbaColor
+    {
+        public byte R { get; set; }
+
+        public byte G { get; set; }
+
+        public byte B { get; set; }
+
+        public byte A { get; set; }
+
+        public RgbaColor(byte r, byte g, byte b)
+            : this(0xFF, r, g, b)
+        {}
+
+        public RgbaColor(byte a, byte r, byte g, byte b)
```

Comment:
Not sure it is good idea to call class `RgbaColor` and then accept components in ARGB order :)

Could you please update that to use RGBA order? Then you'll be able to get rid of second constructor using the default argument value.

Reply:
For some reason, at one point it seemed to me a good idea to make order as in wpf/avalonia

Reply:
Ah, whatever, I like RGBA better :)